src/openepd/model/epd.py

```diff
@@ -27,54 +27,47 @@
 from openepd.model.versioning import OpenEpdVersions, Version
 
 
-class BaseEpd(RootDocument):
+class BaseDeclaration(RootDocument):
     """
     Base class for EPD documents.
 
     This class should not be used directly.  Use Epd or EpdVx instead.
     """
 
-    pass
-
-
-class EpdV0(WithAttachmentsMixin, WithAltIdsMixin, BaseEpd):
-    """Represent an EPD."""
-
-    _FORMAT_VERSION = OpenEpdVersions.Version0.as_str()
-
-    # TODO: Add validator for open-xpd-uuid on this field
     id: str | None = pyd.Field(
         description="The unique ID for this EPD.  To ensure global uniqueness, should be registered at "
         "open-xpd-uuid.cqd.io/register or a coordinating registry.",
         example="1u7zsed8",
         default=None,
     )
-    product_name: str | None = pyd.Field(
-        max_length=200, description="The name of the product described by this EPD", example="Mix 12345AC", default=None
+    date_of_issue: datetime.datetime | None = pyd.Field(
+        example=datetime.datetime(day=11, month=9, year=2019, tzinfo=datetime.timezone.utc),
+        description="Date the EPD was issued. This should be the first day on which the EPD is valid.",
```

Comment:
You've pulled this field up into `BaseDeclaration` but you are still using "EPD" word in the description which will be used for the GenericEstimate.